server/src/controllers/tooljet_db.controller.ts

```diff
@@ -25,64 +24,54 @@ export class TooljetDbController {
 
   @All('/:organizationId/proxy/*')
   @CheckPolicies((ability: TooljetDbAbility) => ability.can(Action.ProxyPostgrest, 'all'))
-  async proxy(@User() user, @Req() req, @Res() res, @Next() next) {
-    return this.postgrestProxyService.perform(user, req, res, next);
+  async proxy(@Req() req, @Res() res, @Next() next, @Param('organizationId') organizationId) {
+    return this.postgrestProxyService.perform(req, res, next, organizationId);
   }
 
   @Get('/:organizationId/tables')
   @UseGuards(TooljetDbGuard)
   @CheckPolicies((ability: TooljetDbAbility) => ability.can(Action.ViewTables, 'all'))
-  async tables(@User() user, @Param('organizationId') organizationId) {
-    const result = await this.tooljetDbService.perform(user, organizationId, 'view_tables');
+  async tables(@Param('organizationId') organizationId) {
+    const result = await this.tooljetDbService.perform(organizationId, 'view_tables');
```

Comment:
We should check
`user.organizationId === organizationId`
ability check is done using `user.organizationId`

Reply:
Same for all routes